core/color/functional.json

```diff
@@ -45,63 +14,95 @@
       "80": "rgba(0, 0, 0, 0.8)",
       "90": "rgba(0, 0, 0, 0.9)",
       "95": "rgba(0, 0, 0, 0.95)",
-      "100": "rgba(0, 0, 0, 1)"
+      "100": "#000000"
```

Comment:
The value of this is in rgba in the Figma file. Why does it appear in hex here?

Reply:
see below comment